src/tests/process/dhcpv6/server.conf

```diff
@@ -0,0 +1,179 @@
+#
+#  test configuration file.  Do not install.
+#
+#  $Id$
+#
+
+#
+#  Minimal dhcpv6.conf for testing
+#
+
+testdir      = $ENV{TESTDIR}
+output       = $ENV{OUTPUT}
+run_dir      = ${output}
+raddb        = raddb
+pidfile      = ${run_dir}/radiusd.pid
+panic_action = "gdb -batch -x src/tests/panic.gdb %e %p > ${run_dir}/gdb.log 2>&1; cat ${run_dir}/gdb.log"
+
+maindir      = ${raddb}
+radacctdir   = ${run_dir}/radacct
+modconfdir   = ${maindir}/mods-config
+certdir      = ${maindir}/certs
+cadir        = ${maindir}/certs
+test_port    = $ENV{TEST_PORT}
+
+#  Only for testing!
+#  Setting this on a production system is a BAD IDEA.
+security {
+	allow_vulnerable_openssl = yes
+}
+
+modules {
+	detail {
+		filename = ${radacctdir}/%{%{Packet-Src-IP-Address}:-%{Packet-Src-IPv6-Address}}/detail-%Y%m%d
+		escape_filenames = no
+		permissions = 0600
+		header = "%t"
+	}
+
+	always reject {
+		rcode = reject
+	}
+	always fail {
+		rcode = fail
+	}
+	always ok {
+		rcode = ok
+	}
+	always handled {
+		rcode = handled
+	}
+	always invalid {
+		rcode = invalid
+	}
+	always disallow {
+		rcode = disallow
+	}
+	always notfound {
+		rcode = notfound
+	}
+	always noop {
+		rcode = noop
+	}
+	always updated {
+		rcode = updated
+	}
+}
+
+#
+#  This virtual server is chosen for processing requests when using:
+#
+#	radiusd -Xd src/tests/ -i 127.0.0.1 -p 12340 -n test
+#
+server dhcpv6 {
+	namespace = dhcpv6
+
+	all_dhcp_servers_and_relays = FF02::1:2
+	interface = eth0
+	port = 547
+
+	listen all_dhcp_servers_and_relays {
+		type = Solicit
+		type = Request
+		transport = udp
+
+		udp {
+			ipaddr = ${...all_dhcp_servers_and_relays}
+			port = ${...port}
+			interface = ${...interface}
+			src_ipaddr = e80::1079:d2fb:1073:74a0
+		}
+	}
+
+	listen local_network {
+		type = Request
+		type = Solicit
+		type = Information-Request
+
+		transport = udp
+
+		udp {
+			ipaddr = 2001:db8::
+			port = ${...port}
+			interface = ${...interface}
+		}
+	}
+
+	dhcpv6 {
+#		status_code_on_success = no
+#		send_failure_message = no
+#		move_failure_message_to_parent = yes
+	}
+
+	recv Solicit {
+		ok
+	}
+
+	send Advertise {
+		&reply += {
+			&Transaction-ID = 0xd81eb8
+			&IA-PD.IAID = 33752069
+			&IA-PD.T1 = 150
+			&IA-PD.T2 = 250
+
+			&IA-PD.Options = {
+				&IA-PD-Prefix.Preferred-Lifetime = 250
+				&IA-PD-Prefix.Valid-Lifetime = 300
+				&IA-PD-Prefix.IPv6-Prefix = 2a00:1:1:100::/56
+			}
+			&Client-ID.DUID = LL
+			&Client-ID.DUID.LL.Hardware-Type = Ethernet
+			&Client-ID.DUID.LL.Hardware-Type.Ethernet.Address = 00:01:02:03:04:05
+			
+			&Server-ID.DUID = LLT
+			&Server-ID.DUID.LLT.Hardware-Type = Ethernet
+			&Server-ID.DUID.LLT.Time = "Nov 21 2012 08:36:00 UTC"
+			&Server-ID.DUID.LLT.Hardware-Type.Ethernet.Address = 00:11:22:33:44:55
+
+			&Preference = 10
+			&DNS-Servers = 2a01::1
+			&AFTR-Name = "aftr-name.mydomain.net"
+		}
+
+		ok
+	}
+
+	recv Request {
+		ok
+	}
+
+	recv Information-Request {
+		ok
+	}
+
+	send Reply {
+		&Transaction-ID = 0x1e291d
```

Comment:
same as above, transaction ID IAID, client-id, server-id should all automatically be copied across by the state machine